packages/react-hooks/docs/0.overview.md

```diff
@@ -42,18 +45,29 @@ const MyInbox = () => {
     inbox.fetchMessages();
   }, []);
 
+  // Updates message.status to "read"
```

Comment:
im not sure "status" here is read/unread.  i think `read` is a timestamp?  but the code returns it as read:boolean?

Reply:
Ahh. You're right. There are issues with our types here.